python/MDSplus/tests/data_case.py

```diff
@@ -116,9 +116,7 @@ def _executeTests(test, a, b, res, **kwargs):
         test('abs(-_a)', abs(-a), res.pop(), **kwargs)
         test('abs1(-_a)', m.ABS1(-a), res.pop(), **kwargs)
         test('abssq(-_a)', m.ABSSQ(-a), res.pop(), **kwargs)
-        rcount = 11
-        acount = 4+rcount
-        if almost:  # some operations only make sence on floating point
+        if almost:  # some operations only make sense on floating point
```

Comment:
The test harness is using two flags: `almost` and `real`.   What are they supposed to do?   The confusion arises from the flag name versus the associated comment.   It appears that `almost` means floating point?

Reply:
Yes, so `almost` eventually gets passed to `assertAlmostEqual(places=almost)` and is the number of decimal places to care about when comparing floating point numbers.

Reply:
Thanks for the explanation.   That eliminates my confusion.